flowauth/frontend/src/PermissionDetails.jsx

```diff
@@ -206,7 +214,7 @@ class PermissionDetails extends React.Component {
       isPermissionChecked,
       permissionIndeterminate
     } = this.state;
-    const { classes, onClick, permitted } = this.props;
```

Comment:
Out of interest: was the presence of `onClick` a bug, or did you have to remove it because of any changes in this PR?

Reply:
Not a bug, just an unused var.